wormhole-connect/src/views/v2/Bridge/ReviewTransaction/index.tsx

```diff
@@ -0,0 +1,341 @@
+          tokenChain: config.sdkConverter.toChainNameV1(receipt.from),
+          tokenId: getWrappedTokenId(sourceTokenConfig),
+          tokenKey: sourceTokenConfig.key,
+          tokenDecimals: getTokenDecimals(
+            config.wh.toChainId(sourceChain),
+            getWrappedTokenId(sourceTokenConfig),
+          ),
+          receivedTokenKey: config.tokens[destToken].key, // TODO: possibly wrong (e..g if portico swap fails)
+          emitterAddress: undefined,
+          sequence: undefined,
+          block: 0,
+          gasFee: undefined,
+          payload: undefined,
+          inputData: undefined,
+          relayerFee,
+          receiveAmount: receiveAmount.data || '',
+          receiveNativeAmount: receiveNativeAmt,
+        }),
+      );
+
+      routeContext.setRoute(sdkRoute);
+      routeContext.setReceipt(receipt);
+
+      dispatch(setSendTx(txId));
+      dispatch(setRedeemRoute(route));
+      dispatch(setAppRoute('redeem'));
+      setSendError('');
+    } catch (e: any) {
+      console.error('Wormhole Connect: error completing transfer', e);
+
+      const [uiError, transferError] = interpretTransferError(e, sourceChain);
+
+      // Show error in UI
+      setSendError(uiError);
+
+      // Trigger transfer error event to integrator
+      config.triggerEvent({
+        type: 'transfer.error',
+        error: transferError,
+        details: transferDetails,
+      });
+    } finally {
+      dispatch(setIsTransactionInProgress(false));
+    }
+  };
+
+  const walletsConnected = useMemo(
+    () => Boolean(sendingWallet.address) && Boolean(receivingWallet.address),
+    [sendingWallet.address, receivingWallet.address],
+  );
+
+  // Review transaction button is shown only when everything is ready
+  const confirmTransactionButton = useMemo(() => {
+    if (
+      !sourceChain ||
+      !sourceToken ||
+      !destChain ||
+      !destToken ||
+      !route ||
+      !(Number(amount) > 0)
+    ) {
+      return null;
+    }
+
+    return (
+      <Button
+        disabled={isTransactionInProgress}
+        variant="contained"
+        color="primary"
+        className={classes.confirmTransaction}
+        onClick={() => send()}
+      >
+        <Typography textTransform="none">
+          {mobile ? 'Confirm' : 'Confirm transaction'}
+        </Typography>
+      </Button>
+    );
+  }, [sourceChain, sourceToken, destChain, destToken, route, amount]);
+
+  if (!route || !walletsConnected) {
+    return <></>;
+  }
+
+  return (
+    <Stack className={classes.container}>
+      <div>
+        <IconButton onClick={() => props.onClose?.()}>
+          <ChevronLeft />
+        </IconButton>
+      </div>
+      <SingleRoute
+        config={RoutesConfig[route]}
+        available={true}
+        isSelected={false}
+        showDestinationGasFee
+        title="You will receive"
+      />
+      <Collapse in={showGasSlider}>
+        <GasSlider
+          destinationGasFee={receiveNativeAmt}
```

Comment:
I don't like calling this a fee. This isn't a fee you are paying, it's an amount you are swapping of the destination token you are bridging, to the destination chain's native gas token. A more appropriate name would be `destinationGasReceiveAmount`.

Reply:
Yup, we have already discussed this below and I have it in my follow-up changes:
https://wormholelabsxyz.slack.com/archives/C06T9FN059C/p1721406582348799